core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -128,12 +128,28 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertTrue(config.useNewConsumer)
+    assertFalse(config.useOldConsumer)
     assertEquals("localhost:9092", config.bootstrapServer)
     assertEquals("test", config.topicArg)
     assertEquals(0, config.partitionArg.get)
     assertEquals(3, config.offsetArg)
     assertEquals(false, config.fromBeginning)
+
+  }
+
+  @Test
+  def testDefaultConsumer() {
+    //Given
```

Comment:
I think @arunmahadevan was following the pattern in the other tests in the class. I agree with you though.